scss/components/_menu.scss

```diff
@@ -22,9 +22,13 @@ $menu-items-padding: $global-menu-padding !default;
 /// @type Number
 $menu-simple-margin: 1rem !default;
 
-/// Text color of an active menu item.
+/// Text color for menu items.
 /// @type Color
-$menu-item-color-active: $white !default;
+$menu-item-color: $white !default;
+
+/// Alternative text color for menu items.
+/// @type Color
+$menu-item-color-alt: $black !default;
```

Comment:
I think we should use `$menu-item-color-active` and `$menu-item-color-alt-active` for backward compatibility and concistency.